### PR TITLE
Small cleanup of README; added failed_votes.md

### PR DESCRIPTION
**Pull request by rknegjens**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,15 +1,17 @@
+# Nomic
```
Comment (akvadrako) [inline README.md]: I don't really want this here - perferably zero headings, but I  could compromise on one if you insist.

File: README.md [hunk 2]
```diff
@@ -1,15 +1,17 @@
...
 ----------- | ----- | ------
 akvadrako   | Ba    | -2
 RamonRB     | Be    | 11
 rknegjens   | K     | 0
 roarandrock | W     | 0
 
-## New Rules
-
-there is nothing here
+## Original rules
```
Comment (akvadrako) [inline README.md]: Again we don't need two levels of headings.

File: failed_rules.md [hunk 3]
```diff
@@ -0,0 +1,3 @@
+## Failed rule proposals
```
Comment (akvadrako) [inline failed_rules.md]: Shouldn't this file be called log or history or something? Since it will include both failed rules and repealed rules?

**Comment by akvadrako** [comment]
I added a history file. How's that?

https://github.com/akvadrako/nomic/blob/master/history.md

File: README.md (hunk 1 above)
Comment (rknegjens) [reply README.md]: Ok you can remove

File: README.md (hunk 2 above)
Comment (rknegjens) [reply README.md]: the point is that Mutable/Immutable and New/Original are separate things. I propose removing the latter then

File: failed_rules.md (hunk 3 above)
Comment (rknegjens) [reply failed_rules.md]: history is fine

**Pull request merged**